cluster-autoscaler/core/scale_down.go

```diff
@@ -912,57 +961,51 @@ func (sd *ScaleDown) scheduleDeleteEmptyNodes(emptyNodes []*apiv1.Node, cp cloud
 				}
 			}()
 
+			deleteErr = waitForDelayDeletion(nodeToDelete, client, sd.context.AutoscalingOptions.NodeDeletionDelayTimeout)
+			if deleteErr != nil {
+				klog.Errorf("Problem with empty node deletion: %v", deleteErr)
+				result = status.NodeDeleteResult{ResultType: status.NodeDeleteErrorFailedToDelete, Err: deleteErr}
```

Comment:
I think this warrants a new result type (e.g. NodeDeleteErrorDelayDeletionAnnotationFailedToDisappear), the FailedToDelete one was supposed to be a failure to delete from cloud provider.

Reply:
Nevermind, discussed in person and this is ok,